docs/src/how_to/parse_matpower_psse.md

```diff
@@ -8,8 +8,41 @@ file_dir = joinpath(pkgdir(PowerSystems), "docs", "src", "tutorials", "tutorials
 sys = System(joinpath(file_dir, "case5.m"))
 ```
 
-This parsing code was copied with permission from
-[`PowerModels.jl`](https://github.com/lanl-ansi/PowerModels.jl).
+Originally, the parsing code was copied with permission from
+[`PowerModels.jl`](https://github.com/lanl-ansi/PowerModels.jl) but over the years the code base
+has had some divergence due to the need to adapt it to for large industrial cases.
+
+The PSSe parser tries to handle correctly all the gotchas that result from the diverse modeling practices transmission engineers employ. However, it is impossible to anticipate all possible variations.
+
+PowerSystems.jl parsing code has been tested and developed using large cases from North America
+like the Western Electricity Coordinating Council (WECC) planning case and the Multiregional Modeling Working Group (MMWG) base case models. This parser has also been adapted to load cases in Latin America and the Caribbean and as many of the open data sets available only.
+
+## [Conventions when parsing MATPOWER or PSS/e Files](@id parse_conventions)
+
+!!! Info
+    
+    In PowerSystems v5, the parsing conventions changed from those in PowerSystems v4. You might experience different behaviors when loading MATPOWER or PSS/e Files.
+
+PowerSystems.jl utilizes a data model that bridges the gap between operational simulations, such as Production Cost Analysis, and electrical engineering simulations, including power flows. Given the different practices in these domains, there are several discrepancies in how to handle data and we have made changes to make the modeling compatible.
+
+In PowerSystems v5, we have implemented the followinf conventions for parsing PSSe files:
+
+  - **BusType correction**: If a bus has a value set to ISOLATED in PSSe, we will confirm that the bus is not entirely disconnected from the network. If the bus is disconnected, it will be set to ISOLATED and set the field available to false. However, if the bus is connected to a generator, we will infer a bus of type PV and set the field 'available' to false. This correction also applies to Matpower. For any other device connected to the bus, we will set it to PQ and set the 'available' field to false. Check [`Understanding ACBusTypes`](@ref bustyped) for a detailed explanation.
```

Comment:
Just clarify that the `available` field of both the bus and the attached generator are set to `false`